packages/ui/src/apis/reset-password.ts

```diff
@@ -6,7 +6,7 @@ type Response = {
 
 export const sendResetPasswordSmsPasscode = async (phone: string) => {
   await api
-    .post('/api/session/reset-password/sms/send-passcode', {
+    .post('/api/session/forgot-password/sms/send-passcode', {
```

Comment:
- also the filename?
- will it be better to use a constant to store the `/api/session/forgot-password` prefix?

Reply:
updated